apps/token/src/lib/view-form-as-json-new-window.ts

```diff
@@ -0,0 +1,21 @@
+export const downloadJson = (jsonString: string, proposalTitle: string) => {
+  const now = new Date();
+  const day = now.getDate().toString().padStart(2, '0');
+  const month = now.toLocaleString('en-US', { month: 'short' });
+  const year = now.getFullYear().toString();
+  const hours = now.getHours().toString().padStart(2, '0');
+  const minutes = now.getMinutes().toString().padStart(2, '0');
+  const seconds = now.getSeconds().toString().padStart(2, '0');
+  // e.g. "2023-Jan-03-23-59-59"
+  const formattedDateTime = `${day}-${month}-${year}-${hours}-${minutes}-${seconds}`;
+
+  const blob = new Blob([jsonString], { type: 'application/json' });
+  const url = URL.createObjectURL(blob);
+  const a = document.createElement('a');
+  a.style.display = 'none';
+  a.href = url;
+  a.download = `${proposalTitle}-${formattedDateTime}.json`;
+  document.body.appendChild(a);
```

Comment:
question (blocking): We aren't cleaning up this link, is this intentional or should we add a 

`a.parentNode.removeChild(a);`

Reply:
not intentional, good spot, thanks